smoke-test/src/main/java/org/opennms/smoketest/containers/OpenNMSContainer.java

```diff
@@ -215,6 +224,21 @@ public OpenNMSContainer(StackModel model, OpenNMSProfile profile) {
         DevDebugUtils.setupMavenRepoBind(this, "/root/.m2/repository");
     }
 
+    public void installFeature(String feature, Path kar) {
+        if (kar != null) {
+            copyFileToContainer(MountableFile.forHostPath(kar),
+                    "/usr/share/opennms/deploy/" + kar.getFileName().toString());
```

Comment:
There is simpler api available to add feature with OpenNMSContainer#getFeaturesOnBoot()

Reply:
`installFeature` installs an arbitrary KAR file and activates the feature. The feature I used this with is `opennms-plugins-cortex-tss` which isn't shipped with OpenNMS, so the existing strategy with `getFeaturesOnBoot` won't help me much (unless I extend it to do the same thing here).

It is used here and in one other place (both are used to install the Cortex TSS plugin):
https://github.com/OpenNMS/opennms/pull/5526/files#diff-ad04b7d08912ca3be37fa1a1737ada52edadf97e021f3d16beaf631a1f69ffbfR69